recipes/r-ukbrapr/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "0.2.9" %}
+{% set sha256 = "693c0ab85d21bea722d6f389898d75eddc3201e07bfb3b3a9e208290dd914d6e" %}
+
+package:
+  name: r-ukbrapr
+  version: {{ version }}
+
+source:
+  - url: https://github.com/lcpilling/ukbrapR/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+  script: R CMD INSTALL --build .
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('r-ukbrapr', max_pin="x") }}
```

Comment:
I don't think this is correct for a `0.x` package.  The package declares lifecycle as 'experimental', which to me connotes low API stability. This should be at least `x.x` or even `x.x.x`.